source/_introduction.html.md.erb

```diff
@@ -298,9 +298,38 @@ The JSON representation of the objects returned from the List All REST endpoints
 Smartsheet will add new functionality and bug fixes to the API over time. Make sure that your code can handle new JSON properties gracefully. 
 Also, make sure your code does not depend on the order in which JSON objects are returned, unless it is explicitly stated in this documentation. 
 
+In some cases, such as when new column types or new widgets are added, Smartsheet offers a backwards-compatibility option as well as giving you access to the new functionality. For instance, if a sheet has column types of MULTI_CONTACT_LIST or MULTI_PICKLIST, there are two ways of receiving the response:
```

Comment:
We offer backwards-compatible responses on RestDataType bumps, I am not sure if all level increases (now and forever) will offer backwards compatible responses.

Reply:
@jordy, do you know if we have committed to always providing backwards compatible responses for GET requests on all level bumps?

Reply:
Would be great to be more specific in regards to backwards-compatibility. 

I know that we will always be backwards compatible on GET requests when we create new data types. 
We will also never provide a backwards compatible POST option